shared/engine/index.js

```diff
@@ -166,7 +166,12 @@ class Engine {
     const handlers = this._onDisconnectHandlers
     // Don't allow mutation while we're handling the handlers
```

Comment:
could this cause new handlers added while this is running to be dropped?

Reply:
no, https://github.com/keybase/client/blob/master/shared/engine/index.js#L432